homeassistant/components/fritz/image.py

```diff
@@ -87,4 +90,6 @@ async def async_image(self) -> bytes:
             self._current_qr_bytes = qr_bytes
             self.async_write_ha_state()
 
-        return qr_bytes
+    def image(self) -> bytes | None:
```

Comment:
This can be async, right?
```suggestion
    async def async_image(self) -> bytes | None:
```

Reply:
yeah, you're right ... not sure, why i changed it 🤦